common/src/chain/config/mod.rs

```diff
@@ -81,6 +90,8 @@ pub struct ChainConfig {
     max_block_header_size: usize,
     max_block_size_with_standard_txs: usize,
     max_block_size_with_smart_contracts: usize,
+    epoch_length: NonZeroU64,
+    sealed_epoch_distance_from_tip: usize,
```

Comment:
Please provide documentation for these new variables... we should document these variables they're becoming way too many.